hcl/hcl.go

```diff
@@ -109,6 +112,7 @@ type DoSync struct {
 	Copies       []Copy        `hcl:"copy,block" json:"copies,omitempty"`
 	DockerLogs   []DockerLog   `hcl:"docker-log,block" json:"docker_log,omitempty"`
 	JournaldLogs []JournaldLog `hcl:"journald-log,block" json:"journald_log,omitempty"`
+	VaultDebugs  []VaultDebug  `hcl:"vaultdebug,block" json:"vaultdebug,omitempty"`
```

Comment:
Looks like the hcl should be `vault-debug` and json should be `vault_debug` for consistency with the other runners.

Reply:
I updated this while Dave is away.